src/components/ActionBar/src/ActionBarButton.vue

```diff
@@ -183,10 +183,43 @@ export default {
 			assert.error(['squared', 'rounded', 'pill'].includes(shapeValue), `${shapeValue} resolved from ${shapeValueOrPointer} is not a valid shape value and cannot be used in the shape prop of ActionBarButton`);
 			return shapeValue;
 		},
+		resolvedTextColor() {
+			let textColorValueOrPointer;
+			if (this.textColor) {
+				textColorValueOrPointer = this.textColor;
+			} else {
+				textColorValueOrPointer = this.theme.actionbarbutton.textColor;
+			}
+			const textColorValue = this.theme.resolve(textColorValueOrPointer);
```

Comment:
For convenience, what do you think about this update to the the `resolve()` method:
```
/**
 * Resolves passed in val which can be a pointer to a theme prop. If val not set, falls back to themePath. 
 * Which will also be resolved if it is apointer.
 * @param {String} themePath
 * @param {*} [val]
 * @return {*} resolvedVal
 * @example
 * this.theme.resolve('actionbarbutton.textColor', this.textColor);
 */
```

It feels overloaded, but worth considering optimizing as this logic will be recurring across the codebase.

Reply:
if you have suggestions for changes on how the `resolve` utility function works please feel welcome to open a PR. I'd prefer to keep this PR focused and scoped to the task of making a few more props on ActionBarButton and Button themeable rather than going on tangentially-related refactor side-quests.

I updated this PR with some JSDoc comments anyway.